src/App.test.jsx

```diff
@@ -5,11 +5,10 @@ test('Should render the header', async () => {
   render(<App />)
 
   const image = screen.getByAltText(/alchemy/i)
-  expect(image).toBeInTheDocument()
-
+  const bannerColor = screen.getByRole('banner', {})
   const profileName = await screen.findByText(/Vonta/i)
```

Comment:
As this is a heading (something inside of an "h" tag), would prefer a getByRole for practice here. Could mock up a user object w/ a name inside  and do something like: 
const user = { name: "Vonta" }
expect(await screen.findByRole('heading', { name: user.name }))